src/main/java/de/tum/in/www1/artemis/service/LectureImportService.java

```diff
@@ -162,7 +162,7 @@ private Attachment cloneAttachment(final Attachment importedAttachment) {
         attachment.setVersion(importedAttachment.getVersion());
         attachment.setAttachmentType(importedAttachment.getAttachmentType());
 
-        Path oldPath = filePathService.actualPathForPublicPathOrThrow(URI.create(importedAttachment.getLink()));
+        Path oldPath = FilePathService.actualPathForPublicPathOrThrow(URI.create(importedAttachment.getLink()));
```

Comment:
There is another call of a static method in FilePathService below. After changing this you could also remove the filePathService attribute in this class.